test/Motor.Extensions.Hosting.RabbitMQ_IntegrationTest/RabbitMQTestBuilder.cs

```diff
@@ -238,6 +261,12 @@ public bool IsConsumerQueueDeclared()
 
         using var channel = Fixture.Connection.CreateModel();
         channel.QueueDeclarePassive(QueueName);
+
+        if (withDeadLetterExchange)
+        {
+            channel.QueueDeclarePassive($"{QueueName}Dlx");
```

Comment:
(nitpick) Maybe introduce a public property for the dead letter queue name so this string doesn't also has to be constructed in the test (`ConsumerStartAsync_ConsumerWithDlxRejectMessage_MessageIsInDlxQueue`)?